src/discrete-layer/components/layers-results/cell-renderer/layer-details.cell-renderer.tsx

```diff
@@ -17,8 +17,9 @@ interface DetailsProp {
 
 const stringRenderer: FormatterFunc = (val): string => val !== undefined ? val.toString() : '';
 const dateRenderer: FormatterFunc = (date): string => {
-  return date !== undefined
-    ? moment(new Date(date.toLocaleString())).format('DD/MM/YYYY HH:mm')
+  // eslint-disable-next-line
+  return (date !== undefined && ("toISOString" in (date as any)))
+    ? moment(date).format('DD/MM/YYYY HH:mm')
```

Comment:
use same time format in all app (config)

Reply:
Done